cmd/domain/root.go

```diff
@@ -1,15 +1,14 @@
-/*
-Copyright © 2023 NAME HERE <EMAIL ADDRESS>
-*/
-package cmd
+package domain
 
 import (
+	"fmt"
+
 	"github.com/spf13/cobra"
 )
 
-// generateCmd represents the generate command
-var generateCmd = &cobra.Command{
-	Use:   "generate",
+// domain/rootCmd represents the domain/root command
+var DomainCmd = &cobra.Command{
```

Comment:
DomainCmd should be fine if it private no? change it to private var if its fine

Reply:
It needs to be public, because this file ```cmd/root.go``` is using this variable.

Reply:
> It needs to be public, because this file `cmd/root.go` is using this variable.

Why dont we just register the domain command inside it? thus we dont need to expose DomainCmd because RootCmd already exposed to all packages.

Reply:
i tried to register the domain inside the ```cmd/domain/root.go``` before, but it doesn't work. The app didn't read the command when i do that. So that, i make the DomainCmd to public. But it should be fine if  RootCmd is private

Reply:
weird enough, ill let it slide for now